index.html

```diff
@@ -9,23 +9,52 @@
   <title>Meet landing page</title>
 </head>
 <body>
-  Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.
+  <div class="container">
```

Comment:
The whole section must be in a container
![Screenshot from 2021-09-21 21-24-33](https://user-images.githubusercontent.com/90243506/134234711-60c0c064-226b-42f6-80e4-df333c01c923.png)
.

Reply:
I think I don't understand you quite well..
My structure is:
open body -> open div container -> header -> main -> footer -> close div container -> close body
When you  say "The whole section must be in a container"
Do you mean that I have to create new container with header and main inside like this:
open body -> open div container -> 
                       open new container -> header -> main -> close new container 
-> footer   ->close div container -> close body
?

Reply:
Your structure is good. All you have to do is make a container that will have some width. See here how this colleague did it.
https://exercise-2-git-develop-obrad97.vercel.app/

Reply:
Is it OK now?
![Screenshot (408)](https://user-images.githubusercontent.com/43373641/134334270-7052288b-756e-4ff0-aa77-64f7d0259174.png)

Reply:
Yes, great!